Lib/ntpath.py

```diff
@@ -647,6 +645,15 @@ def commonpath(paths):
     # Windows XP and non-Windows OS'es will mock _getfinalpathname.
```

Comment:
Python 3.7 dropped Windows Vistaa support: is this test still useful?

https://pythondev.readthedocs.io/platforms.html#windows